docs/source/technical_tutorials/access/accessing_LVIS_data.ipynb

```diff
@@ -0,0 +1,1765 @@
+{
```

Comment:
<p>Not sure if we need this cell as it just grabs 100 collections from NASA CMR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I agree. Removed this cell.</p>